frontend/src/components/ColumnTable.vue

```diff
@@ -79,7 +79,7 @@ import {
   SensitiveData,
   SensitiveDataPolicyPayload,
 } from "@/types";
-import { ColumnMetadata, TableMetadata } from "@/types/proto/database";
+import { ColumnMetadata, TableMetadata } from "@/types/proto/store/database";
```

Comment:
Is there any reason to add the `store` subpath?

Reply:
It's generated by buf for aligning with path.